sdk/core/core/inc/az_http.h

```diff
@@ -129,12 +128,12 @@ typedef enum
 } _az_http_response_kind;
 
 /**
- * @brief az_http_response represents an HTTP response.
+ * @brief Represents an HTTP response.
```

Comment:
Maybe we can improve this a bit to be more meaningful. I am open to suggestions here.

Reply:
Allows you to parse an HTTP response's status line, headers, and body.